File: Python/461.Hamming_Distance.py
class Solution(object):
    def hammingDistance(self, x, y):
        """
        :type x: int
        :type y: int
        :rtype: int
        """
        z = x ^ y
        a = "{0:b}".format(z)
        b = 0
        for i in a:
            if i == "1":
                b += 1
        return b

File: 461.Hamming_Distance.java
class Solution {
    public int hammingDistance(int x, int y) {
        int z = x ^ y;
        String bin_str = Integer.toBinaryString(z);
        int count = 0;
        for(int i = 0; i < bin_str.length();i++){
            if(bin_str.charAt(i) == '1'){
                count ++;
            }
        }
        return(count);
    }
}